pynbody/plot/sph.py

```diff
@@ -364,7 +368,7 @@ def image(sim, qty='rho', width="10 kpc", resolution=500, units=None, log=True,
 
     width = float(width)
 
-    kernel = sph.Kernel()
+    kernel = sph.Kernel(type=kernel_type)
```

Comment:
Here you can create the appropraite kernel object based on the string passed in (i.e. it's ok for the user-facing bit to have strings)